.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+---
+
+## Checklist:
+
+- Does this pull request close an issue? We encourage you to open an issue first if this pull request (PR) is not a
+  minor change.
+  - [ ] No
+    - [ ] The change in this pull request is minor.
```

Comment:
Not sure this should be singular or plural.

Reply:
Either probably works since sometimes a small PR is just one change

Reply:
This is to say that if the answer is no, this checkbox should also be checked.